File: beecrowd/c/bee1132.c
#include <stdio.h>

int main() {

  int x, y, sum = 0;

  scanf("%d %d", &x, &y);

  if (x > y) {
    int temp = y;
    y =  x;
    x = temp;
  }

  for (int i = x; i <= y; i++) {
    if (i % 13 != 0) sum += i;
  }

  printf("%d\n", sum);

  return 0;
}

File: beecrowd/go/bee1132.go
package main

import "fmt"

func main() {
	var x, y, sum int = 0, 0, 0

	fmt.Scanf("%d", &x)
	fmt.Scanf("%d", &y)

	if x > y {
		temp := y
		y = x
		x = temp
	}

	for x <= y {
		if x%13 != 0 {
			sum += x
		}

		x++
	}

	fmt.Printf("%d\n", sum)

}
